docs/src/main/paradox/google-cloud-bigquery-storage.md

```diff
@@ -0,0 +1,146 @@
+It is seen as an improvement over the REST API, and bulk data `extract` jobs for accessing BigQuery-managed table data, but doesn't offer any functionality around managing BigQuery resources.
+Further information at the official [Google Cloud documentation website](https://cloud.google.com/bigquery/docs/reference/storage).
+@@@
+
+This connector communicates to the BigQuery Storage API via the gRPC protocol. The integration between Akka Stream and gRPC is handled by the
+[Akka gRPC library](https://github.com/akka/akka-grpc). Currently this connector only supports returning each row as an Avro GenericRecord.
+@@project-info{ projectId="google-cloud-bigquery-storage" }
+
+## Artifacts
+
+Akka gRPC uses Akka Discovery internally. Make sure to add Akka Discovery with the same Akka version that the application uses.
+
+@@dependency [sbt,Maven,Gradle] {
+  group=com.lightbend.akka
+  artifact=akka-stream-alpakka-google-cloud-bigquery-storage_$scala.binary.version$
+  version=$project.version$
+  symbol2=AkkaVersion
+  value2=$akka.version$
+  group2=com.typesafe.akka
+  artifact2=akka-stream_$scala.binary.version$
+  version2=AkkaVersion
+  group3=com.typesafe.akka
+  artifact3=akka-discovery_$scala.binary.version$
+  version3=AkkaVersion
+}
+
+The table below shows direct dependencies of this module and the second tab shows all libraries it depends on transitively.
+
+@@dependencies { projectId="google-cloud-bigquery-storage" }
+
+## Build setup
+
+The Alpakka Google Cloud BigQuery Storage library contains the classes generated from [Google's protobuf specification](https://github.com/googleapis/java-bigquerystorage/tree/master/proto-google-cloud-bigquerystorage-v1).
+
+@@@note { title="ALPN on JDK 8" }
+
+For use on JDK 8 the ALPN Java agent needs to be set up explicitly.
+
+@@@
+
+### Maven
+
+When using JDK 8: configure your project to use the Java agent for ALPN and add `-javaagent:...` to your startup scripts as described in the @extref:[Akka gRPC documentation](akka-grpc:/buildtools/maven.html#starting-your-akka-grpc-server-from-maven).
+
+### sbt
+
+When using JDK 8: Configure your project to use the Java agent for ALPN and add `-javaagent:...` to your startup scripts.
+
+Pull in the [`sbt-javaagent`](https://github.com/sbt/sbt-javaagent) plugin.
+
+project/plugins.sbt
+: @@snip (/project/plugins.sbt) { #grpc-agent }
+
+Enable the Akka gRPC and JavaAgent plugins on the sbt project.
+
+build.sbt
+: @@snip (/build.sbt) { #grpc-plugins }
+
+Add the Java agent to the runtime configuration.
+
+build.sbt
+:   ```scala
+    javaAgents += "org.mortbay.jetty.alpn" % "jetty-alpn-agent" % "2.0.9"
+    ```
+
+### Gradle
+
+When using JDK 8: Configure your project to use the Java agent for ALPN and add `-javaagent:...` to your startup scripts as described in the @extref:[Akka gRPC documentation](akka-grpc:/buildtools/gradle.html#starting-your-akka-grpc-server-from-gradle).
```

Comment:
This is not required anymore (since JDK 8 update 252).

Reply:
I just merged https://github.com/akka/alpakka/pull/2312 which upgrades Akka gRPC and the Google Pub/Sub gRPC docs with this info.